subjects/java/Buy01/audit/README.md

```diff
@@ -0,0 +1,53 @@
+#### Initial Setup & Access
+
+##### Download the project and initiate it. Try accessing the web pages and the API endpoints using a web browser and tools like Postman.
```

Comment:
Maybe better to tell the auditee to run the web app in a docker container

Reply:
done